modules/system/base.nix

```diff
@@ -5,7 +5,7 @@
     IFS="." read -r -a macOSVersion <<< "$(sw_vers -productVersion)"
 
```

Comment:
Could you add logic here to detect if we have more than one line containing `run\t` and to delete all of the lines?

Also, could you add logic to `chmod` the file to `644` if it is not already that?

Reply:
Most files in `/etc` are world readable and in my opinion this file should be as well, I don't think any of the information in it is secret